src/plugins/legacy.js

```diff
@@ -39,7 +39,6 @@ export type LegacyCommand = {
 }
 
 export function convertFromV5 (c: LegacyCommand) {
-  if (!c.topic) throw new Error('command has no topic')
```

Comment:
I think this was because heroku-apps had a bunk command, it's probably fine to take the guard out now since I think you fixed that